meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -134,7 +134,6 @@ RESIN_CONFIGS[docker] ?= " \
     CONFIG_EXT4_FS_SECURITY=y \
     CONFIG_AUFS_FS=y \
     CONFIG_AUFS_XATTR=y \
-    CONFIG_IP_NF_TARGET_REJECT=y \
     CONFIG_KEYS=y \
```

Comment:
Can this commit be removed and have the change that added this be addressed by a git rebase so as to keep a clean git history?